messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/ConnectionFactoryAttributes.java

```diff
@@ -573,7 +574,8 @@
                 .build();
 
         SimpleAttributeDefinition TRANSACTION = SimpleAttributeDefinitionBuilder.create("transaction", STRING)
-                .setDefaultValue(new ModelNode().set("transaction"))
+                .setDefaultValue(new ModelNode().set(CommonAttributes.XA))
```

Comment:
@ehsavoie @soul2zimate 
The old value 'transaction' does not seem to exist in any of the new allowed values.
CommonAttributes:
```
String LOCAL = "local";
String NONE = "none";
String XA = "xa";
```
I only see this used by PooledConnectionFactoryAdd, so I think that resource will need transformation of whichever of these new values should map to the old one.

Do you agree?

Reply:
No a transformer is not required as we are narrowing the values but I think a corrector is needed to ensure retro compatibility (@soul2zimate )